config/environments/shared_deployment_config.rb

```diff
@@ -69,11 +84,24 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
 
+  # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
+  # the I18n.default_locale when a translation cannot be found).
+  # config.i18n.fallbacks = true
```

Comment:
Unimportant, but this might be nice to lean on for our "let's translate less stuff" push.